source/stylesheets/_refills-styles.scss

```diff
@@ -339,6 +339,20 @@ section.refill {
 
 // Snippet
 
+.snippets > a {
+  background: #38363E;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Properties should be ordered background, border-radius, box-shadow, color, margin, max-width, text-decoration